tortoise/queryset.py

```diff
@@ -1087,8 +1166,7 @@ def _make_query(self) -> None:
                 self.model,
                 (None,),
             )
-            if append_item not in self._select_related_idx:
```

Comment:
This check is not needed since `self._select_related_idx` is always empty at this point.